src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -133,7 +119,7 @@ public String getAdditionalSourceFileLocation() {
     @Override
     List<String> getJavaFXReflectionClassList() {
         List<String> answer = new ArrayList<>(super.getJavaFXReflectionClassList());
-        answer.addAll(javafxReflectionIosClassList);
+        answer.add(javafxReflectionIosClassList);
```

Comment:
why not answer.add("jniconfig-javax-"+configuration.getArchOs()+".json) on the AbstractTargetConfiguration (with an exists check)?

Reply:
Makes sense, I've refactored it.